Major enhancements to documentation

droidmonkey opened the pull request:
Update Fuzz Testing instructions and location
* Also fix env var name

Major updates to documentation:
* Closes #11467 - Describe default search fields
* Closes #11468 - Fix lock database shortcut
* Closes #8259 - Add a note about 1Password OPVault specifics
* Closes #9794 - Add section anchors for easy linking
* Closes #10316 - Show how to setup managed Edge on macOS
* Closes #7805 - Document entry url handling
* Closes #9143 - Document database merge behavior
* Closes #10876 - Correct wording in browser and passkey sections

Update outdated images of the user interface. Improve wording and flow of entire documentation. Fill in missing pieces based on user feedback.

Also updated the demo.kdbx database to add tags and passkey demonstrations.

## Type of change
[NOTE]: # ( Please remove all lines which don't apply. )
- ✅ Documentation (non-code change)

droidmonkey committed:
Update fuzz testing instructions

* Also fix env var name

varjolintu commented on docs/topics/BrowserIntegration.adoc:
```diff
@@ -1,190 +1,192 @@
+image::browser_statistics.png[]
+
+=== Advanced Usage
+You can configure unique browser integration behavior for each entry. This allows you to add multiple URLs to an entry, hide an entry from the browser integration, and more. To access these settings, open an entry for editing then click on _Browser Integration_ option in the left-hand menu *(1)*.
```
Should we mention the new URL wildcard support here?

droidmonkey replied:
Yah, add it in

varjolintu reviewed (approved):
Just double-check the parts I added.

droidmonkey committed:
Major enhancements to documentation

* Closes #11467 - Describe default search fields
* Closes #11468 - Fix lock database shortcut
* Closes #8259 - Add a note about 1Password OPVault specifics
* Closes #9794 - Add section anchors for easy linking
* Closes #10316 - Show how to setup managed Edge on macOS
* Closes #7805 - Document entry url handling
* Closes #9143 - Document database merge behavior
* Closes #10876 - Correct wording in browser and passkey sections

Update outdated images of the user interface. Improve wording and flow of entire documentation. Fill in missing pieces based on user feedback.

Add mention about URL wildcards

droidmonkey committed:
Improve browser notice about snap/flatpak support
